navigator_engine/common/network.py

```diff
@@ -6,67 +6,98 @@
 class Network():
 
     def __init__(self, networkx):
+
```

Comment:
Why putting all those extra empty lines only in this file?

Reply:
Bit by bit, I'm intending to put more white space back into my code with some general rules of thumb:
- All PEP rules apply, as well as...
- One line (or more) above a function declaration, if statement, for statement or any line ending in `:` and a further line line below the corresponding code block
- One line below a function declaration
- One line above a return statement